include/boost/beast/http/impl/read.ipp

```diff
@@ -18,6 +18,7 @@
 #include <boost/beast/core/handler_ptr.hpp>
 #include <boost/beast/core/read_size.hpp>
 #include <boost/beast/core/type_traits.hpp>
+#include <boost/beast/core/coroutine.hpp>
```

Comment:
I don't think this should be public (and it has other problems mentioned earlier) but for future reference when we add new public interfaces, it is best done in its own individual commit so that people who look at the changes will know that there's a new feature. Also, the docs need to be updated.

Reply:
Yeah, you're right, it doesn't belong in the public API. I'll move the is_continuation function to asio instead.